birdhouse/scripts/check-wps-database.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 
+THIS_FILE="`realpath "$0"`"
```

Comment:
Is there a reason why we don't prefer `readlink -f "$0" || realpath "$0"` here as well?

Reply:
No. I simply copy-pasted the definition from other files. I can apply it everywhere if that is preferable.